internal/cli/service.go

```diff
@@ -462,8 +462,17 @@ func cmdService(
 
 	// Wait for termination signal
 	select {
-	case <-sigChan:
-		logger.Infoln("Received SIGTERM, the service is closing")
+	case sig := <-sigChan:
+		var sigName string
+		switch sig {
+		case os.Interrupt:
+			sigName = "SIGINT"
+		case syscall.SIGTERM:
+			sigName = "SIGTERM"
+		default:
+			sigName = sig.String()
```

Comment:
I could also just use this for all cases, but it prints `interrupt` instead of `SIGINT` and `terminated` instead of `SIGTERM`, which might be confusing for some people.